### PR TITLE
pj-22-create-comment

### PR DESCRIPTION
**Pull request by NykoEugen**

### CODE REVIEW

File: comments/tests.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+from django.test import TestCase
+from django.urls import reverse
+
+from comments.forms import CommentForm
+from core.tests import TestDataMixin
+
+
+class CreateCommentTest(TestDataMixin, TestCase):
```
Comment (acman) [inline comments/tests.py:9]: 👍

**Review by acman — COMMENTED** [review]
Looks good! But it would be great to change comment model here

**Comment by NykoEugen** [comment]
need make migrate

File: comments/models.py [hunk 2]
```diff
@@ -1,10 +1,13 @@
 from django.db import models
 
+from core.models import SlugModel
 
-class Comment(models.Model):
+
+class Comment(SlugModel):
```
Comment (acman) [inline comments/models.py]: Comment will not have the detail view, so there is no reason to have slug in Comment. So it should be inherited from model.Model

File: comments/migrations/0003_comment_slug.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Generated by Django 5.0 on 2024-01-07 13:20
```
Comment (acman) [inline comments/migrations/0003_comment_slug.py]: this migration should be removed.

Before removing it!!! Do migrate to 0002, only after it remove slug from model and this migration.

**Review by acman — CHANGES_REQUESTED** [review]
Almost there! `slug` should be removed from comment

**Review by acman — APPROVED** [review]
👍

**Pull request merged**